lib/matplotlib/cbook.py

```diff
@@ -2661,3 +2661,203 @@ def __exit__(self, exc_type, exc_value, traceback):
                     os.rmdir(path)
                 except OSError:
                     pass
+
+
+class _FuncInfo(object):
+    """
+    Class used to store a function
+
+    Each object has:
+    * The direct function (direct)
+    * The inverse function (inverse)
+    * A boolean indicating whether the function
+      is bounded in the interval 0-1 (bounded_0_1), or
+      a method that returns the information depending
+      on this
+    * A callable (check_params) that returns a bool specifying if a
+      certain combination of parameters is valid.
+
+    """
+    def __init__(self, direct, inverse, bounded_0_1=True, check_params=None):
+        self.direct = direct
+        self.inverse = inverse
+
+        if (hasattr(bounded_0_1, '__call__')):
```

Comment:
I think the callable built in function does not exist for python 3.0 or python 3.1, would this be a problem?